bb.edn

```diff
@@ -66,6 +66,9 @@
   dev:validate-plugins-edn
   logseq.tasks.malli/validate-plugins-edn
 
+  dev:validate-config-edn
+  logseq.tasks.malli/validate-config-edn
```

Comment:
This task is handy if we ever want to validate a global or repo config.edn e.g. `bb dev:validate-config-edn /path/to/config.edn`